docs/README.md

```diff
@@ -33,42 +37,736 @@ All output will adhere to the same general JSON structure:
+
+### Files
+`/files` is used for creating or updating a file requires the API to accept a special field allowing for the base64 file data. Beyond that, it must accept POST requests with the multipart-formdata enctype, to allow for easier uploading of file(s), and must accept uploading by chunk, to allow for larger filesizes. @TODO
+
+### Permissions
+`/permissions` is... @TODO WHAT DOES THIS ENDPOINT DO THAT IS SPECIAL?
+
+### Collections
+`/collections` is similar to columns, this endpoint must alter the database schema directly. @TODO
+
+
+
+
+### Create User
+Creates a new user within this instance
+
+```http
+POST /users
+```
+
+#### Body
+The email and password for the new user to be created. Any other submitted fields are optional, but field keys must match column names within `directus_users`.
+
+```json
+{
+  "email": "rijk@example.com",
+  "password": "d1r3ctus",
+  [other user fields, optional]
+}
+```
+
+#### Possible Results
+| Code                     | Description                                        |
+|--------------------------|----------------------------------------------------|
+| 201 Created              | `data`: The created user, including default fields added by Directus  |
+| 400 Bad Request          | `message`: Syntax error in provided JSON    |
+| 422 Unprocessable Entity | `message`: Column doesn’t exist in collection    |
+
+
+
+
+### Get User
+Gets a single user from within this instance
+
+```http
+GET /users/[primary-key-value]
+```
+
+#### Query Parameters
+| Name          | Default   | Description                                      |
+|---------------|-----------|--------------------------------------------------|
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | `data`: Retrieved user<br>`meta`: Depends on requested metadata  |
+| 404 Not Found    | `message`: Item doesn't exist within collection    |
+
+#### Examples
+
+* Return the user with an ID of `1`
+  ```bash
+  curl -g https://api.directus.io/_/users/1
   ```
 
+
+
+
+### Get Users
+Gets Directus users within this instance
+
+```http
+GET /users
+```
+
+#### Query Parameters
+| Name          | Default   | Description                                      |
+|---------------|-----------|--------------------------------------------------|
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+@TODO DOES THIS REALLY NOT ALLOW FOR FILTERING OFFSET ETC?
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | `data`: Array of Directus users<br>`meta`: Depends on requested metadata  |
```

Comment:
I guess a 400 Bad Request is possible too if you mess up the query parameters. Don't think that's implemented though.